pyproject.toml

```diff
@@ -10,7 +10,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.8"
-dcicutils = "^3.14.0"
+dcicutils = "^4.0.1.1b0"
```

Comment:
Hopefully we can get this part merged so that it's not necessary to use a beta here.